pennylane/beta/tapes/tape.py

```diff
@@ -890,6 +906,60 @@ def execute(self, device, params=None):
 
         return self._execute(params, device=device)
 
+    def _get_all_parameters(self, params):
+        """Return all parameters by combining trainable parameters supplied by ``params`` with
+        existing non-trainable parameters.
+
+        The returned parameters are provided in order of appearance
+        on the tape.
+
+        Args:
+            params (list[Any]): The quantum tape operation parameters.
```

Comment:
These would only be parameter values for the trainable parameters, correct? :thinking: Thinking maybe it would be good to further clarify that here and potentially change the name of the object from `params` (`trainable_param_values` comes to mind but perhaps a bit too long? :thinking: )